debian/vzlogger.postinst

```diff
@@ -11,6 +11,7 @@ case "$1" in
 
 	touch /var/log/vzlogger.log
 	chown vzlogger:adm /var/log/vzlogger.log
+	touch /etc/vzlogger.conf
```

Comment:
does creating an empty file make any sense?
(and if the file exists, we should not be messing with it's timestamp?!)

i'm guessing this is only to avoid errors from chmod below?

i think we should either leave the file missing (and the service could detect this and not start - i think some deamons do that at least).
or we should at least provide some content - iirc the shipped file has a set of disabled examples and should work.

Reply:
Makes total sense. The newest version now installs the example vzlogger.conf as a config file to /etc.

Reply:
Unfortunatley I could not test this yet as the build fails with an (probably unrelated) test failure. So I can't resolve this yet.

Reply:
Tested (after also allpying pr #528 ), works now as config file, which is even better!